docs.html

```diff
@@ -404,6 +405,43 @@ <h1 class="display-title">Error Inputs</h1>
               </iframe>
             </div>
           </section>
+          <section class="display-code-content" id="text-utilities">
+            <div class="text-container">
+              <!--  -->
+              <h1 class="heading-bg">Heading BIG</h1>
+              <br>
+              <h2 class="heading-sm">Heading SMALL</h2>
+              <div>
```

Comment:
you may add some more heading tags - like h3,h4....

Reply:
Will do that